.github/workflows/push-oci.yml

```diff
@@ -28,24 +28,22 @@ jobs:
         uses: azure/setup-helm@v4
         with:
           version: v3.14.4
-      - id: releaser
-        name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.6.0
-        env:
-          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-        with:
-          charts_dir: charts
-          config: .github/helm-release-config.yml
-          skip_upload: true
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GH_PKGS_WRITE }}
+      - name: Patch chart name
+        uses: mikefarah/yq@v4.44.2
```

Comment:
I think we have `yq` already available on the ubuntu runner image? https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Reply:
Oh, nice!